docs/en/operation/StressBench.md

```diff
@@ -263,3 +263,532 @@ Here is a sample usage demo, where we have one master and three workers:
+  Each job worker executes one bench job. You can use `--cluster-limit` to specify how many bench jobs run. Cluster throughput is calculated by aggregating the throughput of each bench job.
+
+### Limitations
+- Job Service Stress Bench only supports loading self-generated test files.
+- The `CreateFiles` operation is only used for generating test files instead of measuring throughput.
+
+## Client IO Stress Bench
+
+The Client IO Stress Bench is a tool to measure the IO performance of Alluxio through the client.
+
+### Parameters
+The parameters for the Client IO Stress Bench are (other than common parameters for any stress bench):
+
+<table class="table table-striped">
+    <tr>
+        <th>Parameter</th>
+        <th>Default Value</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>operation</td>
+        <td>Required. No default value.</td>
+        <td>The operation to perform. Options are [ReadArray, ReadByteBuffer, ReadFully, PosRead, PosReadFully]</td>
+    </tr>
+    <tr>
+        <td>client-type</td>
+        <td>AlluxioHDFS</td>
+        <td>The client API type. Alluxio native or hadoop compatible client</td>
+    </tr>
+    <tr>
+        <td>read-type</td>
+        <td>CACHE</td>
+        <td>The cache mechanism during read. Options are [NONE, CACHE, CACHE_PROMOTE]</td>
+    </tr>
+    <tr>
+        <td>clients</td>
+        <td>1</td>
+        <td>The number of fs client instances to use</td>
+    </tr>
+    <tr>
+        <td>threads</td>
+        <td>[1]</td>
+        <td>The comma-separated list of thread counts to test. The throughput for each thread count is benchmarked and measured separately.</td>
+    </tr>
+    <tr>
+        <td>base</td>
+        <td>alluxio://localhost:19998/stress-client-io-base</td>
+        <td>The base directory path URI to perform operations in. </td>
+    </tr>
+    <tr>
+        <td>base-alias</td>
+        <td></td>
+        <td>The alias for the base path, unused if empty</td>
+    </tr>
+    <tr>
+        <td>file-size</td>
+        <td>1g</td>
+        <td>The files size for IO operations. (100k, 1m, 1g, etc.)</td>
+    </tr>
+    <tr>
+        <td>buffer-size</td>
+        <td>64k</td>
+        <td>The buffer size for IO operations. (1k, 16k, etc.)</td>
+    </tr>
+    <tr>
+        <td>block-size</td>
+        <td>64m</td>
+        <td>The block size of files. (16k, 64m, etc.)</td>
+    </tr>
+    <tr>
+        <td>warmup</td>
+        <td>15s</td>
+        <td>The length of time to warmup before recording measurements. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>duration</td>
+        <td>30s</td>
+        <td>The length of time to run the benchmark. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>read-same-file</td>
+        <td>false</td>
+        <td>If true, all threads read from the same file. Otherwise, each thread reads "
+          + "from its own file.</td>
```

Comment:
You don't need string concatenation within HTML. Just wrap the line with a newline and any number of white spaces, browsers take care of the rest.